packages/react-components/review/beta/react-components.api.md

```diff
@@ -498,6 +500,31 @@ export interface DialpadStyles {
     textField?: Partial<ITextFieldStyles>;
 }
 
+// @internal
+export const _DomainPermissions: (props: DomainPermissionsProps) => JSX.Element;
+
+// Warning: (ae-internal-missing-underscore) The name "DomainPermissionsProps" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal
+export interface DomainPermissionsProps {
+    // (undocumented)
```

Comment:
Can we have these props documented?
A small description as to what they do and change would be great.

Reply:
For now they are to be internal so no comment for now?